framework/auth/cas.py

```diff
@@ -208,15 +213,23 @@ def get_logout_url(*args, **kwargs):
     """
     return get_client().get_logout_url(*args, **kwargs)
 
+
 def get_profile_url():
     """Convenience function for getting a profile URL for a user.
     """
     return get_client().get_profile_url()
 
+
```

Comment:
It's really up to personal preference, but according to [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines):

> Surround top-level function and class definitions with two blank lines.
>
>Method definitions inside a class are surrounded by a single blank line.
>
>Extra blank lines may be used (sparingly) to separate groups of related functions. Blank lines may be omitted between a bunch of related one-liners (e.g. a set of dummy implementations).
>
>Use blank lines in functions, sparingly, to indicate logical sections.